app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -639,9 +632,8 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
-  // TODO (#1855): Resolve ktlint max line in app module test
   @Test
-  fun testAudioWithWifi_openFractionsExploration_changeLanguage_clickNext_checkLanguageIsHinglish() { // ktlint-disable max-line-length
+  fun testAudioWithWifi_fractionsExploration_changeLang_next_langIsHinglish() {
```

Comment:
we can short the `Exploration` to `Exp` everywhere

Reply:
@anandwana001 Do I need to change this in the fun which doesn't cross the 100 letter limit also to keep consistency?

Reply:
I think it will be good to short only those where we are crossing 100 char limit, else where keep them as it is